### ISSUE TITLE
Nonfatal bug at snis_server.c:12690

### ISSUE THREAD

**Issue by MCMic** [smcameron/space-nerds-in-space#188]
Starting lobby, then multiverse, then server through snis_launcher all with default settings (so default solarsystem as well), I get these lines in server log:

```
Reading starbase model specifications...done
.//share/snis/solarsystems/default/assets.txt: planet texture ../../textures/planet-texture4- is type earthlike but has no normalmap
.//share/snis/solarsystems/default/assets.txt: planet texture ../../textures/planet-texture4- is type earthlike but has no normalmap
Solarsystem asset spec .//share/snis/solarsystems/default/assets.txt: 0 errors, 2 warnings.
Lua setup done.
Nonfatal bug at snis_server.c:12690
Nonfatal bug at snis_server.c:12690
Nonfatal bug at snis_server.c:12690
Nonfatal bug at snis_server.c:12690
initialize.lua ran
```

The normal map ones were explained in https://github.com/smcameron/space-nerds-in-space/issues/185#issuecomment-459376708 but I did not understand if that meant it should get updated or not.

snis_server.c:12690 seems related to warpgates but I did not understand much more.

**Comment by smcameron** [comment]
It's because NWARPGATES is 10, and NPLANETS is 6.  It's trying to put all the warp gates next to planets, and no more than one warp gate per planet, which is not possible if NWARPGATES > NPLANETS.

Fixed by b9244df7c23b7d78b8816415d63ef3388ac39316

As for the normal maps in the default solar system, I am not planning to update that.  New solarsystem assets go on spacenerdsinspace.com

**Comment by MCMic** [comment]
`Nonfatal bug at snis_server.c:12690` is gone.
